bookkeeper-dist/src/main/resources/LICENSE-all.bin.txt

```diff
@@ -300,11 +300,11 @@ Apache Software License, Version 2.
 - lib/com.google.errorprone-error_prone_annotations-2.1.2.jar [48]
 - lib/org.apache.yetus-audience-annotations-0.5.0.jar [49]
 
-[1] Source available at https://github.com/FasterXML/jackson-annotations/tree/jackson-annotations-2.8.9
-[2] Source available at https://github.com/FasterXML/jackson-core/tree/jackson-core-2.8.9
-[3] Source available at https://github.com/FasterXML/jackson-databind/tree/jackson-databind-2.8.9
-[4] Source available at https://github.com/FasterXML/jackson-modules-base/tree/jackson-modules-base-2.8.4
-[5] Source available at https://github.com/FasterXML/jackson-module-scala/tree/f9e099
+[1] Source available at https://github.com/FasterXML/jackson-annotations/tree/jackson-annotations-2.9.7
+[2] Source available at https://github.com/FasterXML/jackson-core/tree/jackson-core-2.9.7
+[3] Source available at https://github.com/FasterXML/jackson-databind/tree/jackson-databind-2.9.7
+[4] Source available at https://github.com/FasterXML/jackson-modules-base/tree/jackson-modules-base-2.9.7
+[5] Source available at https://github.com/FasterXML/jackson-module-scala/tree/jackson-module-scala-2.9.7
```

Comment:
Do we really need this jar?

Reply:
this is `all` package, it is including all stats providers. that specific dependency is coming from finagle stats provider, I guess it is required for that provider to be working.

Reply:
I see